hw6/Calculator.java

```diff
@@ -0,0 +1,20 @@
+package hw6;
+
+public class Calculator {
+
+    static int addition(int num, int num2) {
```

Comment:
Would be better to use public. It is looks like utility methods

Reply:
done